lib/actions/api.js

```diff
@@ -775,25 +879,6 @@ function createQueryAction (endpoint, responseAction, errorAction, options = {})
   }
 }
 
-// TODO: Determine how we might be able to use GraphQL with the alternative
-// transit index. Currently this is not easily possible because the alternative
-// transit index does not have support for GraphQL and handling both Rest and
-// GraphQL queries could introduce potential difficulties for maintainers.
-// function createGraphQLQueryAction (query, variables, responseAction, errorAction, options) {
-//   const endpoint = `index/graphql`
-//   const fetchOptions = {
-//     method: 'POST',
-//     body: JSON.stringify({ query, variables }),
-//     headers: { 'Content-Type': 'application/json' }
-//   }
-//   return createQueryAction(
-//     endpoint,
-//     responseAction,
-//     errorAction,
-//     { ...options, fetchOptions }
-//   )
-// }
```

Comment:
This might actually be a good approach to use to query the transit index. We could only use it if the config doesn't have an alternate transit index.

Reply:
I wonder now if we even need to support alternate transit index here, since TriMet is no longer using otp-rr.